connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -318,10 +318,17 @@ message Drop {
   // (Required) The input relation.
   Relation input = 1;
 
-  // (Required) columns to drop.
+  // (Optional) columns to drop.
   //
-  // Should contain at least 1 item.
-  repeated Expression cols = 2;
+  // 'columns' and 'column_names' are mutually exclusive.
+  // If set, should contain at least 1 item.
+  repeated Expression columns = 2;
+
+  // (Optional) column names to drop.
+  //
+  // 'columns' and 'column_names' are mutually exclusive.
```

Comment:
Why not make it oneof then?

Reply:
now we need both for python client, since the inputs can contain columns and names together